101-SQL-Injection-Attack-Prevention/nested/microsoft.sql/servers.securityalertpolicies.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "sqlServerName": {
+        "type": "string"
+      },
+      "storageAccountName": {
+        "type": "string",
+        "defaultValue": "storageaccountname"
```

Comment:
remove this or use a uniqueString to generate a unique name

Reply:
this needs to be a mandatory parameter. lets not assume any default value for it. i am removing it.